bitnami/spark/templates/secret.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
This should be generated only when I do not include my own secret. See the existingSecret option in charts like PostgreSQL

Reply:
You are right. Done.

Reply:
Ok

Reply:
Ok